pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: tizen_interop
-description: Provides Dart bindings and utility functions for Tizen native APIs.
-version: 0.0.1
+description: Provides Dart bindings and utility functions for using Tizen native APIs.
+version: 0.1.0
 homepage: https://github.com/flutter-tizen/tizen_interop
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
 
 dependencies: 
-  ffi: ^1.1.2
+  ffi: ">=1.1.2 <3.0.0"
```

Comment:
For better compatibility (see https://github.com/flutter-tizen/plugins/pull/427).